src/components/Contacts/ContactDetails/ContactNotesTab/ContactNotesTab.graphql

```diff
@@ -0,0 +1,25 @@
+query GetContactNotes($contactId: ID!, $accountListId: ID!) {
+  contact(id: $contactId, accountListId: $accountListId) {
+    id
+    notes
+    notesSavedAt
+  }
+}
+
+mutation UpdateContactNotes(
+  $contactId: ID!
+  $accountListId: ID!
+  $notes: String!
+) {
+  updateContact(
+    input: {
+      accountListId: $accountListId
+      attributes: { id: $contactId, notes: $notes }
+    }
+  ) {
+    contact {
+      id
+      notes
```

Comment:
Can you add `notesSavedAt` to the mutation response too? That way when we save the notes, we fetch the new date. Since Apollo updates the cache automatically for use, the `useQuery` will update and rerender the component when the cache updates and show the new date.

Reply:
I thought this was the magic you were talking about yesterday. When the notes get updated, the `notesSavedAt` field is updated automatically and the output text re-renders. No?